website/content/docs/security/security-models/core.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
-page_title: Consul Core Security Model
+page_title: Security Models: Consul Core
 description: >-
-  Security model including requirements, recommendations, and threats for the core Consul product.
+The security model for Consul Core details requirements and recommendations for securing access to your network. Learn about potential threats and how to protect Consul from malicious actors.
```

Comment:
AFAIK this page doesn't cover general network access recommendations, only those specific to Consul.
```suggestion
The security model for Consul Core details requirements and recommendations for securing your deployment of Consul. Learn about potential threats and how to protect Consul from malicious actors.
```